contributors/guide.md

```diff
@@ -54,6 +54,29 @@ Additional Capabilities:
 
 Contribution to this project goes beyond the notion of code contributions. We value any help that you may be able to provide.
 
+#### Component
+
+Components developed outside of the Cloud Native Buildpacks project can be accepted following this process:
```

Comment:
Feedback:

1. This is a large wall of text for a type of contribution that is bound to happen less frequently.
2. The contents are pretty much verbaidum to that of the RFC.

Given that this document is more intended to new contributors would it be easier on the eyes if we simply had a link to the RFC. For example:

```suggestion
Components developed outside of the Cloud Native Buildpacks project can be accepted following the process defined in [this RFC](https://github.com/buildpacks/rfcs/blob/main/text/0086-component-level-contrib.md#what-it-is).
```